pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -21,70 +28,58 @@ import (
 func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command {
 	opts := &Options{}
 	inspectCmd := &cobra.Command{
-		Use:    "inspect [<file path> | oci://<OCI image URI>] --bundle <path-to-bundle>",
-		Args:   cmdutil.ExactArgs(1, "must specify file path or container image URI, as well --bundle"),
+		Use:    "inspect <path-to-sigstore-bundle>",
+		Args:   cmdutil.ExactArgs(1, "must specify bundle file path"),
 		Hidden: true,
-		Short:  "Inspect a sigstore bundle",
+		Short:  "Inspect a Sigstore bundle",
 		Long: heredoc.Docf(`
 			### NOTE: This feature is currently in public preview, and subject to change.
 
-			Inspect a downloaded Sigstore bundle for a given artifact.
+			Inspect a Sigstore bundle that has been downloaded to disk. To download bundles
+			associated with your artifact(s), see the %[1]sdownload%[1]s command.
```

Comment:
**Should this be more explicit?**

Just to separate this from `gh run download` or `gh release download`:

```suggestion
			associated with your artifact(s), see the %[1]sgh at download%[1]s command.
```